tests/test_app.py

```diff
@@ -1,37 +1,33 @@
+"""
```

Comment:
Need to reinstate all unit tests temporarily removed as otherwise we'll lose them and our coverage rates.
Apart from that LGTM